### ISSUE TITLE
Couple of minor UX suggestions

### ISSUE THREAD

**Issue by testphreak** [anshooarora/extentreports#149]
You might already know this or have it in your roadmap, but it would be nice -
1. If you can wrap long titles of tests in the test details pane to the next line. Right now I need to scroll to see the full title of the test once I select it in the left pane. The left pane actually wraps the test name, so that's good.
2. Would be nice if you allow the test view edge (between the test view and the test details) to be dragged to the left or right.

<img width="1428" alt="screen shot 2015-10-19 at 8 25 19 pm - feedback" src="https://cloud.githubusercontent.com/assets/10567800/11013126/c92adb64-84b5-11e5-90ad-0e26c6bebf63.png">

**Comment by anshooarora** [comment]
1. fixed
2. Will have to research this.. there are a few plugins that allow doing this but that would require adding more scripts to the page which will make it heavier (slower).  I would ideally like as few external js resources as possible.

**Comment by testphreak** [comment]
I'll take performance anytime :). I agree about not adding more scripts to the page. I was wondering if we could use CSS3 to do the job for us. Essentially resize the divs on the page the way we want.

http://www.w3schools.com/cssref/tryit.asp?filename=trycss3_resize

**Comment by ssirekumar** [comment]
Sorry if i interrupt in thread, For this why don't we add like collapse/expand manner 
http://jsfiddle.net/rbxL7/5/   here we have vertical bar to drag to left or right. Here our test case list will be drag to left. If the user drag to left the test case result detail will be expand..

**Comment by anshooarora** [comment]
css resize only allows re-sizing to the right.  We would require something like this:  http://gridster.net/

I think I know of a way to do it without using external libs - will have to code and test once v2.40 is stable.

**Comment by testphreak** [comment]
This is minor Anshoo, if it can be done for 2.40 great :)!

**Comment by MegaUkrainian** [comment]
Hi,

Meanwhile, if its not take a lot of time, please help how to adjust the proportions  permanently for views described in the the 2nd UI suggestion via JS injection.
I do not need resizing. I just want to make test details view a little a bit bigger.

Thank  you in advance.

**Comment by anshooarora** [comment]
Here are the 2 elements that control the containers:

```
Test Names -> "#test-view > .col.s5"
Test Details -> "#test-details-wrapper > .details-view"
```

You can play around with their widths to see what works best..

**Comment by MegaUkrainian** [comment]
Thanks, works good

**Comment by anshooarora** [comment]
@testphreak @ssirekumar - The ability to resize containers will be available soon.

**Comment by testphreak** [comment]
Awesome! Thanks Anshoo!

**Comment by anshooarora** [comment]
How does [this](http://relevantcodes.com/Tools/ExtentReports2/samples/149.html) look?

**Comment by testphreak** [comment]
Looks good to me.

**Comment by MegaUkrainian** [comment]
> How does this look?
It's better to hide the scroll if it's unnecessary and give a possibility of resizing on whole the line of delimitation.

**Comment by ssirekumar** [comment]
Yaahaa it's look good but if we consider @MegaUkrainian comments.

**Comment by testphreak** [comment]
Anshoo, not quite sure what MegaUkrainian meant, but I think you've done enough to meet the goals of this enhancement. The only thing I wasn't clear from your example link was the usability experience when there are say 1000 tests and you need to scroll to the bottom to be able to resize the container. I was hoping we could place the cursor anywhere to the right edge of the test-view container and resize it.

**Comment by MegaUkrainian** [comment]
> I was hoping we could place the cursor anywhere to the right edge of the test-view container and resize it.
It's what I exactly meant =)

**Comment by anshooarora** [comment]
Each container has its own scrollbar depending upon the amount of content available.

Latest:  http://relevantcodes.com/Tools/ExtentReports2/samples/149-2.html#!

**Comment by testphreak** [comment]
Sounds good to me. Latest example looks good.

**Comment by sazib** [comment]
Can we add custom filter option? If possible then how?

**Comment by testphreak** [comment]
@sazib custom filter option enhancement is not related to this thread. Let's not use this issue as a catch all for all UI requests. Please create a new one, so it's easier to track.

**Comment by anshooarora** [comment]
Can this be closed?  All reqs here are covered in 2.40.1.

**Comment by testphreak** [comment]
Sure thing! Thanks Anshoo!
